src/libcore/ptr/mod.rs

```diff
@@ -1534,7 +1534,7 @@ impl<T: ?Sized> *const T {
     /// `usize::max_value()`.
     ///
     /// The offset is expressed in number of `T` elements, and not bytes. The value returned can be
-    /// used with the `offset` or `offset_to` methods.
+    /// used with the `offset` method.
```

Comment:
Not mentioning `offset_to` is a good point, but that makes me realize that this returns `usize` but `offset` wants `isize`.  Maybe it would make sense to refer to [<*const T>::add](https://doc.rust-lang.org/std/primitive.pointer.html#method.add) instead?

Reply:
Sounds reasonable. Should I mention both `add` and `offset` or only `add`?

Reply:
@scottmcm I pushed c9c73f5d2550f00e9b952e65d05421721b73ee6d to mention `add` instead of `offset`. Let me know if you prefer keeping the mention of `offset`.